api_app/analyzers_manager/migrations/0067_update_misp.py

```diff
@@ -0,0 +1,96 @@
+from django.db import migrations
+
+
+def migrate(apps, schema_editor):
+    PythonModule = apps.get_model("api_app", "PythonModule")
+    Parameter = apps.get_model("api_app", "Parameter")
+    PluginConfig = apps.get_model("api_app", "PluginConfig")
+
+    pm = PythonModule.objects.get(
+        module="misp.MISP", base_path="api_app.analyzers_manager.observable_analyzers"
+    )
+
+    p1 = Parameter(
+        name="timeout",
+        type="int",
+        description="set timeout for misp instance",
+        is_secret=False,
+        required=True,
+        python_module=pm,
+    )
+    p1.full_clean()
+    p1.save()
+
+    p2 = Parameter(
+        name="published",
+        type="bool",
+        description="get only published events",
+        is_secret=False,
+        required=True,
+        python_module=pm,
+    )
+    p2.full_clean()
+    p2.save()
+
+    p3 = Parameter(
+        name="metadata",
+        type="bool",
+        description="have lighter queries but less data",
+        is_secret=False,
+        required=True,
+        python_module=pm,
+    )
+    p3.full_clean()
+    p3.save()
+
+    for analyzer in pm.analyzerconfigs.all():
+        pc1 = PluginConfig(
+            value=5,
+            analyzer_config=analyzer,
+            for_organization=False,
+            owner=None,
+            parameter=p1,
+        )
+        pc1.full_clean()
+        pc1.save()
+
+        pc2 = PluginConfig(
+            value=False,
+            analyzer_config=analyzer,
+            for_organization=False,
+            owner=None,
+            parameter=p2,
+        )
+        pc2.full_clean()
+        pc2.save()
+
+        pc3 = PluginConfig(
+            value=False,
+            analyzer_config=analyzer,
+            for_organization=False,
+            owner=None,
+            parameter=p3,
+        )
+        pc3.full_clean()
+        pc3.save()
+
+
+def reverse_migrate(apps, schema_editor):
+    PythonModule = apps.get_model("api_app", "PythonModule")
+
+    pm = PythonModule.objects.get(
+        module="misp.MISP", base_path="api_app.analyzers_manager.observable_analyzers"
+    )
+    pm.analyzerconfig.all().delete()
+    pm.parameter.all().delete()
+    pm.delete()
```

Comment:
This `reverse_migration` function it is not actually the reverse of the forward migration

Reply:
okay so is there a way to verify what exactly the reverse migration function does? 
i tried applying and unapplying the migration to verify the functionality in the latest commit.

Reply:
What you described it is the best way. 
Apply the migration -> check the state -> reverse the migration -> check the state

If the question was about if there is some automatically check to verify these steps, the answer unfortunately it is not.
Or at least I do not know a way, we could maybe implement something but I think it is overkill at the moment

Reply:
makes sense, understood